core/storage/classifier/gae_models.py

```diff
@@ -95,19 +100,19 @@ def create(
                 data used by the classifier.
 
         Returns:
-            ID of the new ClassifierModel entry.
+            ID of the new ClassifierDataModel entry.
 
         Raises:
             Exception: A model with the same ID already exists.
         """
 
         instance_id = cls._generate_id(exp_id)
```

Comment:
@prasanna08 , Then why would instance_id = job_request_id? If we are generating a random hash inside the create() function, we shouldn't be passing job_request_id to it then right? The instance_id that is created by _generate_id() is the job_request_id? Or is this random hash generated in the TrainClassifierjobModel's storage layer and just passed here?